index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```

Comment:
den Titel würde ich auf jeden Fall noch ändern
```suggestion
    <title>Gökhan's Profil</title>
```